### PR TITLE
Organizes Web and Cordova docs

### PR DESCRIPTION
**Pull request by scotthasbrouck**

### CODE REVIEW

**Comment by Anu2g** [comment]
Nice job on the documentation, very helpful.

File: CORDOVA_GUIDE.md [hunk 1]
```diff
@@ -28,164 +19,402 @@ This deep link routing callback is called 100% of the time on init, with your li
...
+1. Click the "+" sign to add a new URI Scheme, as below:
 
-**[Formerly `getLatestReferringParams()`](CORDOVA_UPGRADE_GUIDE.md)**
+![URL Scheme Demo](https://s3-us-west-1.amazonaws.com/branchhost/urlType.png)
+
+_____
+
+### SDK Method Queue
+
+Initializing the SDK is an asynchronous method with a callback, so it may seem as though you would need to place any method calls that will execute immidiatley inside the `branch.init()` callback. We've made it even easier than that, by building in a queue to the SDK! The only thing that is required is that `branch.init()` is called prior to any other methods. All SDK methods called are gauranteed to : 1. be executed in the order that they were called, and 2. wait to execute until the previous SDK method finishes. Therefore, it is 100% allowable to do something like:
```
Comment (dmitrig01) [inline CORDOVA_GUIDE.md]: few typos :-)

Comment (dmitrig01) [inline CORDOVA_GUIDE.md]: immidiatley, gauranteed

Comment (dmitrig01) [inline CORDOVA_GUIDE.md]: also i would say even 'recommended' isntead of allowable :-)

**Comment by scotthasbrouck** [comment]
@dmitrig01 added a new step to the Makefile that runs that compiles the Table of Contents at the top of both guides from comments in branch.js. I kept forgetting to update it. see build_utils/toc_generator.pl

**Comment by dmitrig01** [comment]
wow nice perl!! @hubt would be proud

**Comment by scotthasbrouck** [comment]
haha, last time I wrote this much perl, it went into the cgi-bin folder :-P

**Comment by dmitrig01** [comment]
great work. i think this is good to go :+1:

**Pull request merged**